aws/logs_monitoring/trace_forwarder/cmd/trace/main.go

```diff
@@ -145,9 +145,16 @@ func sendTracesToIntake(tracePayloads []*pb.TracePayload) error {
 			fmt.Printf("Failed to send traces with error %v\n", err)
 			hadErr = true
 		}
+		stats := apm.ComputeAPMStats(tracePayload)
+		err = edgeConnection.SendStats(context.Background(), stats, 3)
```

Comment:
The context gets bubbled down to `sendPayloadToTraceEdge` as-is, and there's no timeout on the the request. Should we set a timeout on this context?